src/chainstate/stacks/boot/costs.clar

```diff
@@ -212,6 +210,9 @@
 (define-read-only (cost_sqrti (n uint))
     (runtime u1))
 
+(define-read-only (cost_log2 (n uint))
+    (runtime u1))
```

Comment:
should this be constant or linear based on the number of bits?

Reply:
This should be constant -- there's a size limit for our integers.